utilities/cmake/simulator/simulator.cmake

```diff
@@ -38,6 +38,9 @@ endif()
 target_include_directories(${PROJECT_NAME} PRIVATE
         src/
 
+        ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
What is the need to add `${CMAKE_CURRENT_BINARY_DIR)` into the include path?

Reply:
the generated files are at the root of cmake build directory hence need to provide that for including headers.

Reply:
i tried `src/../` in place of `${CMAKE_CURRENT_BINARY_DIR}` but did not work. The generated protobuf headers could not be located.

Reply:
The example provided by nanopb (ref `vendor/nanopb/extra/FindNanopb.cmake#63`) mentions to use `CMAKE_CURRENT_BINARY_DIR` cmake variable for including protoc generated headers.